biotracks/plot.py

```diff
@@ -34,21 +34,6 @@
 import seaborn as sns
 
 
-def prepareforplot(df, x, y, t):
-    """Prepare data for visualizations.
-
-    Keyword arguments: (passed through command line for the moment)
-    df -- the trajectories dataframe
-    x -- the x_coordinate header
-    y -- the y_coordinate header
-    t -- the time header
-    """
-    x_coord = x
-    y_coord = y
-    time = t
-    df.sort_values(time, axis=0, inplace=True)
-
-
```

Comment:
It's a one-liner (the three assignments are unnecessary) and it's used only once, so I just replaced that single function call with the sort itself.